pkg/controller/compliancescan/compliancescan_controller.go

```diff
@@ -413,16 +457,18 @@ func gatherResults(r *ReconcileComplianceScan, instance *compv1alpha1.Compliance
 	var result compv1alpha1.ComplianceScanStatusResult
 	compliant := true
 	isReady := true
-	for _, node := range nodes.Items {
+
+	switch instance.Spec.ScanType {
+	case compv1alpha1.ScanTypePlatform:
 		targetCM := types.NamespacedName{
-			Name:      getConfigMapForNodeName(instance.Name, node.Name),
+			Name:      getConfigMapForNodeName(instance.Name, "api-checks"),
```

Comment:
I would prefer to avoid using hardcoded constants and define the name in `config.go` (or add `constants.go` if you think it is cleaner)

Reply:
will do